packages/phpactor.rb

```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Phpactor < Package
+  description 'PHP completion, refactoring and introspection tool.'
+  homepage 'https://phpactor.readthedocs.io/en/develop/'
+  version '0.15.0'
+  source_url 'https://github.com/phpactor/phpactor/archive/0.15.0.tar.gz'
+  source_sha256 '02f63e77aa435874a3e8ef11a293eb24a7ee259317dfaae3f058d8bc24e39e24'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'composer'
+  depends_on 'php73'
+
+  def self.preinstall
+  end
+
+  def self.build
+  end
+
+
+  def self.install
+    system "composer install"
+    FileUtils.mkdir_p "#{CREW_PREFIX}/share/phpactor"
+    FileUtils.cp_r '.', "#{CREW_PREFIX}/share/phpactor/."
+    FileUtils.ln_s "#{CREW_PREFIX}/share/phpactor/bin/phpactor", "#{CREW_PREFIX}/bin/phpactor"
```

Comment:
This should be:
```
    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/phpactor"
    FileUtils.cp_r '.', "#{CREW_DEST_PREFIX}/share/phpactor/."
    FileUtils.ln_s "#{CREW_PREFIX}/share/phpactor/bin/phpactor", "#{CREW_DEST_PREFIX}/bin/phpactor"
 ```
Otherwise, crew will not be able to track the install.  Run `crew files phpactor` afterwards and you will know what I mean.  Also, probably need to copy all the files in `$HOME/.config/composer` to `#{CREW_DEST_HOME}/.config/composer` as well due to the `composer install`.

Reply:
Right !! I'll check what's in $HOME

Reply:
Come to think of it, you likely don't need to copy anything from `$HOME/.config/composer` since this isn't a global install...I think.

Reply:
Done. Indeed, nothing created in `$HOME/.config/composer`